pkg/oc/cli/cmd/login/loginoptions.go

```diff
@@ -316,16 +318,21 @@ func (o *LoginOptions) gatherProjectInfo() error {
 			return err
 		}
 		o.Project = current.Name
-
-		fmt.Fprintf(o.Out, "You have access to the following projects and can switch between them with '%s project <projectname>':\n\n", o.CommandName)
-		for _, p := range projects.List() {
-			if o.Project == p {
-				fmt.Fprintf(o.Out, "  * %s\n", p)
-			} else {
-				fmt.Fprintf(o.Out, "    %s\n", p)
+		
+		// Surpress project listing if the number of projects available to the user is greater than the threshold. Prevents unnecessarily noisy logins on clusters with large numbers of projects
+		if len(projectsItems) > projectsItemsSurpressThreshold {
+			fmt.Fprintf(o.Out, "You have access to %d projects, the list has been surpressed. You can list all projects with '%s projects'\n", len(projectsItems), o.CommandName)
```

Comment:
nit: have this end with two newlines to match:

normal user:

```
Login successful.

You have access to the following projects and can switch between them with 'oc project <projectname>':

    abb
    abbb
  * abbbb

Using project "abbbb".
```

admin:

```
Logged into "https://10.13.129.85:8443" as "system:admin" using existing credentials.

You have access to 123 projects, the list has been surpressed. You can list all projects with 'oc projects'
Using project "abbbb".
```

Reply:
I removed that second newline while validating the Fprintf line with @juanvallejo ; should I add it back in?